### ISSUE TITLE
Test Failed Due to Printing Method

### ISSUE THREAD

**Issue by aldoportillo** [DPI-WE/ruby-gym-bank-account#1]
Got the following error when using pp to print to the console as opposed to puts. 

```bash
F

Failures:

  1) BankAccount rejects negative or zero deposits
     Failure/Error: expect { account.deposit(-50) }.to output("Invalid deposit amount\n").to_stdout
     
       expected block to output "Invalid deposit amount\n" to stdout, but output "\"Invalid deposit amount\"\n"
       Diff:
       @@ -1 +1 @@
       -Invalid deposit amount
       +"Invalid deposit amount"
       
     # (test):1:in `block (2 levels) in '

Finished in 0.00986 seconds (files took 0.08922 seconds to load)
1 example, 1 failure

Failed examples:

rspec (test):-1 # BankAccount rejects negative or zero deposits
```

**Comment by heratyian** [comment]
fixed here 96c92d

**Comment by heratyian** [comment]
fixed 96c92d1f95d384df028b233e137524fdbd337313
